client/components/App.jsx

```diff
@@ -10,21 +11,49 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      id: 1,
-      product_name: 'Founders Fresh Metal Golf Clubs Fairway Woods Graphite Shaft Pick Flex & Loft',
-      price: '$78099.96',
-      seller: 'AmerisourceBergen Corporation (Holding Co)',
-      rating: 3,
+      id: 3,
+      product_name: '',
+      price: '',
+      condition: false,
+      rating: 0,
+      review_count: 0,
+      availability: 0,
+      sale_count: 0,
+      seller: '',
+      sellerRating_count: 0,
+      seller_feedback: '',
```

Comment:
re-organized state and changed some of the key names to match up with key names in db.